lib/tui/tui.ml

```diff
@@ -29,10 +29,10 @@ let read_root_tree ~root_dir_path =
   let tree = Fs.read_tree root_dir_path in
   let files =
     match tree with
-    | Fs.File (path, _, _) ->
+    | Fs.File { name = path; _ } ->
         Printf.eprintf "Given path '%s' is not a directory!" path;
```

Comment:
I'm actually fine if you don't use `name = path` and just use `name`. Makes the code consistent with others.

I now think that `path` might not be the most accurate variable here.